### PR TITLE
Adding referal for wimel

### PR DESCRIPTION
**Pull request by derfredy**

### CODE REVIEW

**Comment by RiccardoM** [comment]
I'm sorry but the user `kwunyeung` has already been referred. Also, you should not create files under the `referred` folder as they should be created by the referred people to accept your referral. 

Please remove such files and the last entry inside your referral file

File: phases/phase-1/challenges/referred/kwunyeung [hunk 1]
```diff
@@ -0,0 +1 @@
+derfredy
```
Comment (RiccardoM) [inline phases/phase-1/challenges/referred/kwunyeung]: Please remove this

File: phases/phase-1/challenges/referred/mariopino [hunk 2]
```diff
@@ -0,0 +1 @@
+derfredy
```
Comment (RiccardoM) [inline phases/phase-1/challenges/referred/mariopino]: Please remove this

File: phases/phase-1/challenges/referred/wimel [hunk 3]
```diff
@@ -0,0 +1 @@
+derfredy
```
Comment (RiccardoM) [inline phases/phase-1/challenges/referred/wimel]: Please remove this

**Comment by derfredy** [comment]
> I'm sorry but the user `kwunyeung` has already been referred. Also, you should not create files under the `referred` folder as they should be created by the referred people to accept your referral.
> 
> Please remove such files and the last entry inside your referral file

I am sorry, I misunderstood

**Comment by derfredy** [comment]
If I can not add Kwun to my list...
How should I ask Kwun friendship?

**Comment by RiccardoM** [comment]
> If I can not add Kwun to my list...
> How should I ask Kwun friendship?

Currently, friendship are not supported right now. However, we plan to implement them and release a new challenge for them so stay tuned 😉

**Comment by RiccardoM** [comment]
> > I'm sorry but the user `kwunyeung` has already been referred. Also, you should not create files under the `referred` folder as they should be created by the referred people to accept your referral.
> > Please remove such files and the last entry inside your referral file
> 
> I am sorry, I misunderstood

Perfect! 🎉 Merging!

**Pull request merged**